src/pages/HomePage.js

```diff
@@ -0,0 +1,77 @@
+import Heading from '../components/Header'
+import PropTypes from 'prop-types'
+import ConcertCard from '../components/ConcertCard'
+import styled from 'styled-components/macro'
+
+HomePage.propTypes = {
+  pageName: PropTypes.string,
+  onNavigate: PropTypes.func,
+  bookmarks: PropTypes.array,
+}
+
+export default function HomePage({ bookmarks, pageName, onNavigate }) {
+  return (
+    <Wrapper>
+      <Heading pageName={pageName} />
+      <Hint>
+        <BookmarkInactive>☆</BookmarkInactive>
+        <h2>Merkliste</h2>
+      </Hint>
+      <div>
+        {bookmarks.length === 0 ? (
+          <Text>Du hast Dir noch keine Konzerte gemerkt.</Text>
+        ) : (
+          <ConcertList>
+            {bookmarks.map(concert => (
+              <ConcertCard
+                key={concert.id}
+                date={concert.shortDate}
+                artist={concert.artist}
+                location={concert.location}
+                image3x2={concert.image3x2}
+                price={concert.price}
+                onClick={() => onNavigate(concert.id)}
+              />
+            ))}
+          </ConcertList>
+        )}
+      </div>
+    </Wrapper>
+  )
+}
+
+const Hint = styled.div`
+  margin: 75px 10px 10px;
+  display: flex;
+  align-items: center;
+  gap: 10px;
+`
+
+const Wrapper = styled.section`
+  padding: 10px;
+`
+
+const ConcertList = styled.ul`
```

Comment:
The Concert list is a ul but then the ConcertCard is a section. I am not sure if this is sematic html